java/pom.xml

```diff
@@ -382,6 +382,7 @@
                                       failonerror="true"
                                       executable="cmake">
                                     <arg value="${basedir}/src/main/native"/>
+                                    <arg line="${cmake.ccache.opts}"/>
```

Comment:
Will this do the right thing when `cmake.ccache.opts` is undefined, or will it add an additional empty argument that will potentially confuse cmake?

Reply:
good catch. verified that it works with empty opts now